### ISSUE TITLE
Allow Network Connectivity Status Indicator (msftncsi.com)

### ISSUE THREAD

**Issue by crazy-max** [crazy-max/WindowsSpyBlocker#8]
Windows check a Microsoft site for connectivity, using the Network Connectivity Status Indicator site.

* NCSI performs a DNS lookup on www.msftncsi.com, then requests http://www.msftncsi.com/ncsi.txt. This file is a plain-text file and contains only the text Microsoft NCSI.
* NCSI sends a DNS lookup request for dns.msftncsi.com. This DNS address should resolve to 131.107.255.255. If the address does not match, then it is assumed that the internet connection is not functioning correctly.

If you want to implement your own NCSI, [check this link](http://blog.superuser.com/2011/05/16/windows-7-network-awareness/)
More info : https://technet.microsoft.com/en-us/library/cc766017%28WS.10%29.aspx

**Comment by crazy-max** [comment]
Solved on 2.7

**Comment by h1dden-da3m0n** [comment]
Sorry that I have to revive this one, BUT in order to get the MS Connectivity Indicator to behave we would need "dns.msftncsi.com" unblocked too :( (its included your [extra.txt](https://raw.githubusercontent.com/crazy-max/WindowsSpyBlocker/master/data/hosts/extra.txt) list and can cause issues)

Sidenotes: 
1# THANK YOU FOR THIS PROJECKT! 👍 🥇 
2# I hope you are okay with ppl using your blocklists as pi.hole blocklists too, as this error kind of originated from me using them for that purpose. (sorry if you are not fine with that :-( )

**Comment by h1dden-da3m0n** [comment]
Sorry that I have to revive this one, BUT in order to get the MS Connectivity Indicator to behave we would need "dns.msftncsi.com" unblocked too :( (its included your [extra.txt](https://raw.githubusercontent.com/crazy-max/WindowsSpyBlocker/master/data/hosts/extra.txt) list and can cause issues)

Sidenotes: 
1# THANK YOU FOR THIS PROJECKT! 👍 🥇 
2# I hope you are okay with ppl using your blocklists as pi.hole blocklists too, as this error kind of originated from me using them for that purpose. (sorry if you are not fine with that :-( )

[Repost, sorry again]

**Comment by crazy-max** [comment]
Hi @h1dden-da3m0n, extra rules must be used carefully. Some domains are "required" by third party applications on Windows (Skype, Live, MSFT Connect Test, etc...). But you can use our custom NCSI probe : https://github.com/crazy-max/WindowsSpyBlocker/wiki/appTelemetryNCSI
And thanks for your support :)

**Comment by crazy-max** [comment]
@h1dden-da3m0n @ribatamu I was abale to reproduce your issue.
@ribatamu Can you create a new issue about this please ? Thanks

**Comment by crazy-max** [comment]
See #128
